cpp/src/arrow/vendored/double-conversion/utils.h

```diff
@@ -102,6 +102,7 @@ int main(int argc, char** argv) {
     defined(__AARCH64EL__) || defined(__aarch64__) || defined(__AARCH64EB__) || \
     defined(__riscv) || \
     defined(__or1k__) || defined(__arc__) || \
+    defined(__loongarch__) || \
```

Comment:
Could you open a pull request to upstream https://github.com/google/double-conversion ?
We'll update our bundled version after this change is merged in upstream.

Reply:
https://github.com/google/double-conversion/pull/162/

Reply:
Thanks.

Then could you open a new separated issue to update vendored double-conversion and open a separated pull request for it?
For example, #14817 is a pull request to update vendored fast_float.